compiler/Options.hs

```diff
@@ -58,6 +58,7 @@ data Options
         { files :: [FilePath]
         , import_dir :: [FilePath]
         , output_dir :: FilePath
+        , using :: [String]
```

Comment:
gbc-tests will unconditionally try to `using options`, which will fail if it's a Java options. Rather than create code paths for "all target languages except Java", I just added it back in.